src/js/edu-benefits/utils/validations.js

```diff
@@ -136,52 +46,10 @@ function isValidRoutingNumber(value) {
   return false;
 }
 
-function isValidValue(validator, value) {
-  return isBlank(value) || validator(value);
-}
-
-function isValidField(validator, field) {
-  return isValidValue(validator, field.value);
-}
-
-function isValidRequiredField(validator, field) {
-  return isNotBlank(field.value) && validator(field.value);
-}
-
-function isBlankDateField(field) {
-  return isBlank(field.day.value) && isBlank(field.month.value) && isBlank(field.year.value);
-}
-
 function isBlankMonthYear(field) {
```

Comment:
I wasn't sure about this one, it seems to only be used by education for now. It's possible it will be used for other applications in the future, so I can move it out.